proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -296,89 +300,90 @@ service Microgrid {
   // If any of the checks mentioned above fails, then the method call returns an
   // error.
   //
-  // If any of the above mentioned actions for a given component has already
-  // been performed, then this method call effectively skips that action.
+  // If any of the above mentioned actions for a given electrical component has
+  // already been performed, then this method call effectively skips that
+  // action.
   //
   // If a feature required to perform an action is missing, then that action is
   // skipped.
-  rpc StopComponent(StopComponentRequest) returns (google.protobuf.Empty) {
+  rpc StopElectricalComponent(StopElectricalComponentRequest)
+    returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      get : "/v1/components/{component_id}/stop"
+      get : "/v1/components/{electrical_component_id}/stop"
     };
   }
 
-  // Acknowledges any recoverable error reported by the component, and brings it
-  // back to the stopped or cold-standby state.
-  rpc AckComponentError(AckComponentErrorRequest)
+  // Acknowledges any recoverable error reported by the electrical component,
+  // and brings it back to the stopped or cold-standby state.
+  rpc AckElectricalComponentError(AckElectricalComponentErrorRequest)
     returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      get : "/v1/components/{component_id}/errorAck"
+      get : "/v1/components/{electrical_component_id}/errorAck"
     };
   }
 }
 
 // Metadata that describes a microgrid.
 message GetMicrogridMetadataResponse {
   // The location of the microgrid, in geographical co-ordinates.
-  frequenz.api.common.v1.microgrid.Microgrid  microgrid= 1;
+  frequenz.api.common.v1alpha7.microgrid.Microgrid  microgrid= 1;
 }
 
-// Request parameters for the RPC `ListComponents`.
-// Contains filtering parameters for listing components.
-message ListComponentsRequest {
-  // Return components that have the specified IDs only.
-  repeated uint64 component_ids = 1;
+// Request parameters for the RPC `ListElectricalComponents`.
+// Contains filtering parameters for listing electrical components.
+message ListElectricalComponentsRequest {
+  // Return electrical components that have the specified IDs only.
+  repeated uint64 electrical_component_ids = 1;
```

Comment:
Since this got so long now, I wonder if we really want to have the extra verbosity in request/responses, or maybe be more succinct and use just `ids`, `categories` and `components` in the fields instead of the full `electrical_component_` prefix, as *electrical component* is already in the `Message` and `rpc` names.